src/pages/shared/Like.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { HeartOutline, HeartSharp } from 'react-ionicons';
+import styled from 'styled-components';
+import ReactTooltip from 'react-tooltip';
+import { postLikeOrDislike } from '../../service/service.posts';
+
+export default function Like ({liked, setLiked, id, userInfo, likes}) {
+
+	function likeOrDislike(){
+		const action = (liked) ? 'dislike' : 'like';
+		const config = {headers: 
+			{ 'Authorization': `Bearer ${userInfo.token}` }
+		};
+		setLiked(!liked);
+		postLikeOrDislike(config, id, action).then().catch(()=>{
+			setLiked(!liked);
+		});
+	}
+
+	const peopleWhoLiked = likes.filter((l)=>{if (l.userId!==userInfo.userId) return l.username;});
+	function dataTip(){
+		if (!peopleWhoLiked.length){
+			return (liked) ? 'You' : 'Nobody';
```

Comment:
to sem saber se vamos deixar tudo em pt-br ou tudo em en. a depender, tenho que editar aqui

Reply:
Deixa já assim :)